<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>632</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1182</width>
    <height>632</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1182</width>
    <height>632</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SISTEMA ESTOQUE-ARPROTECT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1171</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/img/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1182</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEstoque">
    <property name="title">
     <string>Esto&amp;que</string>
    </property>
    <addaction name="cadastrar_produto"/>
    <addaction name="consultar_produto"/>
   </widget>
   <widget class="QMenu" name="menuRelat_rio">
    <property name="title">
     <string>Re&amp;latório</string>
    </property>
    <addaction name="relatorio"/>
   </widget>
   <widget class="QMenu" name="menuUsuarios">
    <property name="title">
     <string>&amp;Usuarios</string>
    </property>
    <addaction name="gerenciar_usuario"/>
   </widget>
   <widget class="QMenu" name="menuConfigura_es">
    <property name="title">
     <string>&amp;Configurações</string>
    </property>
    <addaction name="bancodados"/>
    <addaction name="backup"/>
   </widget>
   <widget class="QMenu" name="menuForncedor">
    <property name="title">
     <string>Fornecedor</string>
    </property>
    <addaction name="cadastrar_fornecedor"/>
    <addaction name="consultar_fornecedor"/>
   </widget>
   <addaction name="menuEstoque"/>
   <addaction name="menuForncedor"/>
   <addaction name="menuRelat_rio"/>
   <addaction name="menuUsuarios"/>
   <addaction name="menuConfigura_es"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="cadastrar_produto">
   <property name="text">
    <string>Cadastrar(CTRL+C)</string>
   </property>
  </action>
  <action name="consultar_produto">
   <property name="text">
    <string>Consultar(CTRL+&amp;L)</string>
   </property>
  </action>
  <action name="relatorio">
   <property name="text">
    <string>&amp;Consulta(CTRL+I)</string>
   </property>
  </action>
  <action name="gerenciar_usuario">
   <property name="text">
    <string>Gerenciar(CTRL+U)</string>
   </property>
  </action>
  <action name="actionConsultar_2">
   <property name="text">
    <string>Consultar</string>
   </property>
  </action>
  <action name="bancodados">
   <property name="text">
    <string>&amp;Banco de Dados</string>
   </property>
  </action>
  <action name="backup">
   <property name="text">
    <string>Ba&amp;ckup</string>
   </property>
  </action>
  <action name="actionFornecedor_CTRL_F">
   <property name="text">
    <string>&amp;Fornecedor(CTRL+F)</string>
   </property>
  </action>
  <action name="cadastrar_fornecedor">
   <property name="text">
    <string>Cadastrar(CTRL+F)</string>
   </property>
  </action>
  <action name="consultar_fornecedor">
   <property name="text">
    <string>Consultar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
